cmd/yace/main.go

```diff
@@ -90,7 +89,7 @@ func main() {
 func NewYACEApp() *cli.App {
 	yace := cli.NewApp()
 	yace.Name = "Yet Another CloudWatch Exporter"
-	yace.Version = version
+	yace.Version = version.Version
```

Comment:
IIUC promu will read the version from the `VERSION` file? This is really handy at release time, but is misleading when building an unreleased version. Previously we had the makefile setup so that it would output the current branch and revision, something like this:

```makefile
GIT_BRANCH   ?= $(shell git rev-parse --abbrev-ref HEAD)
GIT_REVISION ?= $(shell git rev-parse --short HEAD)
VERSION      ?= $(GIT_BRANCH)-$(GIT_REVISION)
```

How could we reproduce a similar behaviour?

Reply:
I think this would be a feature request for promu. The version package vars are injected by the `.promu.yml` file.